MusiciansAPP/MusiciansAPP.API/Utils/PagingHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MusiciansAPP.API.Configs;
+
+namespace MusiciansAPP.API.Utils
+{
+    public class PagingHelper
+    {
+        public static int GetCorrectPageSize(int pageSize)
+        {
+            int maxSize = AppConfigs.MaxPageSize;
+            int size = pageSize;
```

Comment:
You don't actually need this variable.
You can slightly simplify the method returning the size without initializing the variable.